# Process Information Message
string name
int32 pid
float32 cpu_percent
float32 memory_percent
